docs/source/getting-started/installing.adoc

```diff
@@ -6,96 +6,46 @@ include::variables.adoc[]
 :toc: macro
 :toc-title:
 :toclevels: 1
+:next-steps: Proceed to the xref:../getting-started/quickstart.adoc#[{project} Quickstart] once {project} has been installed.
 
 toc::[]
 
-[[installing-overview]]
-== Overview
+[[installing-manually]]
+== Manually
```

Comment:
@robin-owen For upstream we do not have any options other than manual install, so I do not think we should mention manual installation. It is the default way for installation.

Reply:
I understand that we had the text before. But do we want to continue with it?

Reply:
@LalatenduMohanty Right, the name was changed here to contrast it with the other heading of "With Homebrew". Considering both headings have identical placement within the hierarchy of the document, it seemed appropriate to label them with what each section describes.

Whether manual installation is the default or not, that's what this portion of the documentation describes.